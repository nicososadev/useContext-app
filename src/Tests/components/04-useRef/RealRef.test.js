import React from 'react'
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import { RealRef } from '../../../components/04-useRef/RealRef'


describe('Pruebas en el componente RealRef', () => {
    
    const wrapper = shallow( <RealRef />)

    test('Debe mostrarse correctamente', () => {
        
        expect( wrapper ).toMatchSnapshot()
    })

    test('Debe mostrar el componente MultipleHooks', () => {

        wrapper.find('button').simulate('click')
        
        expect( wrapper.find('MultipleHooks').exists() ).toBe( true )
    })
    
    
})
