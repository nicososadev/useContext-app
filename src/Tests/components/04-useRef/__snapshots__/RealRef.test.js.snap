// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente RealRef Debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Caso de uso de useRef
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
     Mostrar/Ocultar 
  </button>
</div>
`;
